torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2068,7 +2102,8 @@ def _create_symbolic_sizes_strides_storage_offset(
             }
             # iterate over unbound strides in sorted order
             val_list = sorted(
-                [(ex_stride[i], i) for i in range(len(stride)) if stride[i] is None]
+                [(ex_stride[i], i) for i in range(len(stride)) if stride[i] is None],
+                key=lambda x: (sys.maxsize, x[1]) if is_singleton(x[0]) else x
```

Comment:
Aren't there supposed to just not be any strides in the jagged tensor case? Or, if there are strides, they're symbolic stuff like `4 * j0` but these should be sortable, no?

Reply:
Right, they're symbolic stuff like `4 * j0` but they don't have an ordering

Reply:
If you factor out the j0, you should have an ordering. And you would order things with a j0 factor higher than others.

Reply:
alright I'll try it out

Reply:
Splitting the array into two parts and separately calling sort on the elements that have j0 factor, and one that doesn't should work since comparing `a * j0` with `b * j0` is fine today, but `a * j0` with `c` is not defined. 

> And you would order things with a j0 factor higher than others.

Why is it okay to order it this way (since its usually undefined) in the context of this sorting here?

Reply:
In this particular case, I know it's OK because jagged tensors are always contiguous, so once I multiply in j0 I want that to be bigger than all the non-j0 strides to the right. In fact, I think I'm pretty comfortable claiming this always is true.

Reply:
Gotcha, jagged tensors are indeed contiguous today, though folks are asking for transpose support now...so it may change in the future.